be/src/util/thread_group.h

```diff
@@ -61,7 +61,7 @@ class ThreadGroup {
 
     template <typename F>
     std::thread* create_thread(F threadfunc) {
-        std::lock_guard<std::shared_mutex> guard(_mutex);
+        WriteLock wrlock(_mutex);
```

Comment:
Why do you change `std::lock_guard<std::shared_mutex>` to `std::unique_lock<std::shared_mutex>` ? I think unique_lock may be slower than lock_guard.

Reply:
already fix it